app/assets/javascripts/discourse/app/instance-initializers/post-decorations.js

```diff
@@ -156,7 +156,16 @@ export default {
       function generateFullScreenTableModal(event) {
         const table = event.currentTarget.parentElement.nextElementSibling;
         const tempTable = table.cloneNode(true);
-        modal.show(FullscreenTableModal, { model: { tableHtml: tempTable } });
+        const cookedWrapper = document.createElement("div");
```

Comment:
Is the wrapper element necessary or can those classes and the `data-` attr be set on the cloned `table` element itself? 😃

Reply:
It may be necessary because somewhere we may choose to make some specific css only effective for elements under `.cooked`, and the table is actually extracted directly from the cooked content. It will be more robust under `.cooked`.

For example, there is such code in the core:
```css
.cooked a, .d-editor-preview a {
    word-wrap: break-word;
}
```
In the original case, this CSS cannot be applied to the full-screen table.